### PR TITLE
Updated the loading indicator

### PR DESCRIPTION
**Pull request by YangQi007**

### CODE REVIEW

**Review by anderspitman — CHANGES_REQUESTED** [review]
@YangQi007 rather than duplicating the indicator code, would it be possible to make it into an `iobio-loading-indicator` web component? Assuming you're using the same one everywhere...

**Comment by anderspitman** [comment]
Converting SVG to PNG might prove rather difficult. And it could start to bloat the app size as well. Since those images are controlled by the index.html and not the iobio-charts library, I would expect them to work fine. Can you open another PR that shows the problem?

File: loading_indicator.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        }
+    }
+
+    updateLabel() {
+        const labelElement = this.shadowRoot.querySelector('.label');
+        labelElement.textContent = this.getAttribute('label') || 'Default Label';
+    }
+}
+
+customElements.define('loading-indicator', LoadingIndicator);
```
Comment (anderspitman) [inline loading_indicator.js:64]: Can you rename the registered component to `iobio-loading-indicator`?

**Review by anderspitman — CHANGES_REQUESTED** [review]
Looks great overall. Just one name change request

**Comment by anderspitman** [comment]
@YangQi007 don't forget to request a review when you're ready for me to re-check PRs. Pretty sure you were done with this one so I merged. Great work!

**Comment by YangQi007** [comment]
> @YangQi007 don't forget to request a review when you're ready for me to re-check PRs. Pretty sure you were done with this one so I merged. Great work!

Sorry, I forgot to request a review again. I always thought it would send notifications automatically.

**Comment by anderspitman** [comment]
It does send notifications of commits pushed, but I don't know if you're finished until you request review. For example you might just be pushing commits to back up your work before lunch but planning to push more commits later before requesting review.

**Pull request merged**